.travis.yml

```diff
@@ -4,8 +4,8 @@ matrix:
       dist: xenial
       sudo: required
       language: node_js
-      node_js: 10
-      python: 2.7.13
+      node_js: 16
+      python: 3.10.1
```

Comment:
Let's build with Python 3.7 or 3.8. #.10 is really bleeding edge and too recent IMHO.

Reply:
ok, is 3.7 good?

Reply:
yes :+1: